### PR TITLE
Synchronize Persian language with the latest base version

### PR DESCRIPTION
**Pull request by MrTechHunter**
style: synchronize Persian language with the latest base version

### CODE REVIEW

File: packages/docusaurus-theme-classic/codeTranslations/fa.json [hunk 1]
```diff
@@ -59,4 +59,4 @@
   "theme.tags.tagsListLabel": "برچسب ها:",
   "theme.tags.tagsPageLink": "مشاهده تمام برچسب ها",
   "theme.tags.tagsPageTitle": "برچسب ها"
-}
+}
```
Comment (lex111) [inline packages/docusaurus-theme-classic/codeTranslations/fa.json:62]: Let's add new line at end of file as was before

```suggestion
}

```

**Review by lex111 — COMMENTED** [review]
Also try naming your PR according to our commit style convention (you've read the contributing guidelines, right? https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#semantic-commit-messages)

File: packages/docusaurus-theme-classic/codeTranslations/fa.json (hunk 1 above)
Comment (MrTechHunter) [reply packages/docusaurus-theme-classic/codeTranslations/fa.json]: https://github.com/facebook/docusaurus/pull/5640/commits/c7cce5f6ca031ade49369ba6f7a27a4c7e7567fa

You're so helpful 😊

**Review by lex111 — APPROVED** [review]

**Comment by lex111** [comment]
@MrTechHunter great, thanks! :+1:

**Pull request merged**